internal/handshake/aead.go

```diff
@@ -0,0 +1,56 @@
+package handshake
+
+import (
+	"crypto/cipher"
+	"encoding/binary"
+
+	"github.com/lucas-clemente/quic-go/internal/protocol"
+)
+
+type sealer struct {
+	iv   []byte
+	aead cipher.AEAD
+
+	nonceBuf []byte
```

Comment:
Why is this a member and not a local var?

Reply:
This avoids allocating the `nonceBuf` again and again, for every packet that we send.

Reply:
I added a comment.